wagtail/tests/settings.py

```diff
@@ -91,12 +91,12 @@
     'wagtail.tests.snippets',
     'wagtail.tests.routablepage',
     'wagtail.tests.search',
-    'wagtail.contrib.wagtailstyleguide',
-    'wagtail.contrib.wagtailsitemaps',
-    'wagtail.contrib.wagtailroutablepage',
-    'wagtail.contrib.wagtailfrontendcache',
-    'wagtail.contrib.wagtailapi',
-    'wagtail.contrib.wagtailsearchpromotions',
+    'wagtail.contrib.styleguide',
+    'wagtail.contrib.sitemaps',
+    'wagtail.contrib.routablepage',
+    'wagtail.contrib.frontendcache',
+    'wagtail.contrib.api',
+    'wagtail.contrib.searchpromotions',
```

Comment:
I've never quite worked out what the most Pythonic way to name modules is regarding underscores. Even Django is inconsistent with this: `context_processors` vs `templatetags` for one example. However, Django does not use underscores for any of its `contrib` modules. If Wagtail started using underscores to be more Pythonic, we would become less Djangoish at the same time.

I don't necessarily object to adding underscores if there is a solid argument for it, but I am not convinced it is more Pythonic given the confusion around underscores in module names that is already present.